ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -260,7 +267,7 @@ export class Send extends IronfishCommand {
       )} plus a transaction fee of ${CurrencyUtils.renderIron(
         fee,
         true,
-      )} to ${to} from the account ${from}`,
+      )} to ${to} from the account "${from}" with the memo "${memo}"`,
```

Comment:
Is it okay that this might sometimes display `with the memo ""`?

Reply:
Well, it's true.

I don't feel one way or the other, but I do feel like this should be changed to be more list-like instead of a sentence, if we want to improve it.

Something like:
"You are about to send the following transaction:
Amount: 1234
Fee: 0.00001
Memo: ""
To: asdf12345"

Reply:
Makes sense, let's leave it as is :+1: